packages/exchange/src/exchange/providers/utils.py

```diff
@@ -10,6 +11,27 @@
 from tenacity import retry_if_exception
 
 
+def get_env_url(key: str, default: str = "") -> httpx.URL:
```

Comment:
the default imo should be `None` so it matches the signature

```python
[ins] In [1]: import os
         ...:
         ...: os.environ.get??
Signature: os.environ.get(key, default=None)
Docstring: D.get(k[,d]) -> D[k] if k in D, else d.  d defaults to None.
File:      ~/<frozen _collections_abc>
Type:      method
```

Reply:
removed this